Applied "title3" Classes to Headings.

RyanCahela opened the pull request:
Fixes #2089

### What changes did you make and why did you make them?

  - added the 'title3' utility class to decouple the font size from HTML heading tag.
  - changed 'h3' tags to 'h2' because there were no h2 elements on the page and these headings mark the second most important pieces of information in the document.
  - changed hero image src path to begin with "../" instead of "./" because it wasn't loading with "./" prefix. 

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
<!-- Note, if your images are too big, use the <img src="" width="" length="" />  syntax instead of ![image](link) to format the images -->
<!-- If images are not loading properly, you might need to double check the syntax or add a newline after the closing </summary> tag -->


<details>
<summary>Visuals before changes are applied</summary>

![Before-Full-iPad](https://user-images.githubusercontent.com/6194542/159995625-e3c7eb86-e1af-4c19-8dc3-c349323e6010.png)
![Before-Full-iPhone-X](https://user-images.githubusercontent.com/6194542/159995626-14a2c15b-802a-41b6-9108-aaa4f712c3f9.png)
![Before-Full-Generic-Laptop](https://user-images.githubusercontent.com/6194542/159995618-b2267e92-7a27-4703-8d78-6d2dded374d1.png)
</details>

<details>
<summary>Visuals after changes are applied</summary>
  
![after-desktop](https://user-images.githubusercontent.com/6194542/160941002-47870363-e955-4309-a93b-544928e9141f.png)
![after-tablet](https://user-images.githubusercontent.com/6194542/160941011-dad2c57f-655d-4775-b588-e975651fbf16.png)
![after-mobile](https://user-images.githubusercontent.com/6194542/160941012-440efa6c-8b29-47bd-b585-2bc3c845eb7e.png)


</details>

RyanCahela committed:
removed 'day-heading' class added 'title3' class to headings, changed h3 tags to h2 for better semantics

RyanCahela committed:
changed src path for header-hero-image

tamara-snyder commented:
Availability: 1 hour
Review ETA: Within the next hour

tamara-snyder commented:
Hi Ryan, great work on this! I can see you've changed the class of all the weekdays to ".title3" and the headings to h2 tags, which I think makes more sense in this context. The changes are visible on the page and everything still looks good in both mobile and desktop. Well done!

tamara-snyder reviewed (dismissed):
Hi Ryan, great work on this! I can see that you've changed the weekdays to exist within h2 tags with a class of ".title3", which I think makes more sense in this context. The changes are visible on the page and everything still looks good in both mobile and desktop. Well done!

Also, thank you for fixing the image source. Even though it's not part of the issue, I think it's worth leaving that in. I'll wait for someone else to review it to get a second opinion before merging.

SAUMILDHANKAR reviewed (changes requested):
@RyanCahela Hi Ryan, great job in adding the title3 class and changing h3 tags to h2, however, not sure if we should remove day-header class, since it changes the margin and makes the page look slightly different. I would recommend keeping it for now. 
Also, not sure if we should change the path in this issue, it might be confusing for QA and there might already be a pending issue for it. I would encourage to look if there is already an issue to fix this and if not create one. Thanks again for your detailed analysis. Please request a re-review once you have made changes or let me know if you have questions.

kathrynsilvaconway commented:
ETA: Within one hour.
Availability:
One Hour

RyanCahela committed:
added day-header class for margin

RyanCahela commented:
@SAUMILDHANKAR I've added the 'day-header' class back to give the headings some margin. I left the image path bug fix in per our discussion this morning. 

Let me know if I need to change anything else.

kathrynsilvaconway reviewed (approved):
Nice work updating @RyanCahela the heading tags to h2 from h3. I think this improves the hierarchy and makes the flow of the content a little clearer. Adding the '.title3' class to these tags improves clarity as well.

SAUMILDHANKAR reviewed (dismissed):
@RyanCahela Thanks for making the changes as per team discussion. Code looks all good only requesting minor changes in your initial comment. I understand the "visuals after changes are applied" in your initial comment, would change because of this so requesting to update them as well. Also, it would be really helpful if we could update the initial post where it says changed day header class to title3. Really appreciate your insights on this issue as well as quick turnaround.

RyanCahela commented:
@SAUMILDHANKAR I updated the "after changes" screenshots and changed the first bullet point to reflect adding the "title3" class instead of replacing the "day-header" class. 

Let me know if I need to fix anything else.

JessicaLucindaCheng reviewed (approved):
@RyanCahela Thanks for making the requested changes. It looks good to me.
